src/components/ResultItem/ResultItem.tsx

```diff
@@ -1,17 +1,43 @@
 import React from 'react';
 
+type meta = {
+  type?: string;
+  date?: Date;
+  lastUpdated?: Date;
+};
+
 type ResultItemProps = {
   className?: string;
   href: string;
-  meta?: string;
+  meta?: meta;
   text?: string;
   title: string;
 };
 
-export const ResultItem = ({ meta, title, href, text }: ResultItemProps) => (
+export const ResultItem = ({
+  meta: { type, date, lastUpdated },
+  title,
+  href,
+  text
+}: ResultItemProps) => (
   <div className="result-item">
     <div className="result-item__meta">
-      <span className="result-item__meta-type">{meta}</span>
+      <span className="result-item__meta-type">{type}</span>
+      <span className="result-item__meta-type--date">
```

Comment:
rename this to `result-item__meta-date` since it's not a variant of the type e.g. 'Report' but an element of its own